### PR TITLE
feat(foosball-api): add swagger support

### PR DESCRIPTION
**Pull request by mschilling**
* add Swagger Support: https://docs.nestjs.com/openapi/introduction
* tweak types/models: https://docs.nestjs.com/openapi/types-and-parameters

### CODE REVIEW

**Comment by mschilling** [comment]
Output:

Screenshot 2022-05-19 at 08 47 00

File: apps/foosball/api/src/bootstrap.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { SwaggerModule, DocumentBuilder } from '@nestjs/swagger';
+import { INestApplication } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+
+export function configure(app: INestApplication): void {
```
Comment (mschilling) [inline apps/foosball/api/src/bootstrap.ts:5]: I like to configure (bootstrap) Swagger in a standalone file.

File: apps/foosball/api/src/bootstrap.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  const config = app.get(ConfigService);
```
Comment (mschilling) [inline apps/foosball/api/src/bootstrap.ts:6]: This is a nice feature in Nest :-)

File: apps/foosball/api/src/app/app.controller.ts [hunk 3]
```diff
@@ -1,7 +1,13 @@
 import { Controller, Get } from '@nestjs/common';
+import { ApiProperty, ApiResponse } from '@nestjs/swagger';
 
 import { AppService } from './app.service';
 
+class AppVersion {
+  @ApiProperty()
```
Comment (mschilling) [inline apps/foosball/api/src/app/app.controller.ts:7]: You can annotate properties that you want to include in the docs

**Review by sanderelderman — APPROVED** [review]

**Review by cassshh — APPROVED** [review]

**Pull request merged**